pom.xml

```diff
@@ -527,6 +528,20 @@
                 <artifactId>maven-dependency-plugin</artifactId>
                 <version>3.3.0</version>
             </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <version>5.1.1</version>
+                <executions>
```

Comment:
Should we add something like this to lock-in that the only public-use package is `com.jcraft.jsch` (similar to what I did in `module-info.java` for JPMS)?
```
                <configuration>
                    <instructions>
                        <Export-Package>com.jcraft.jsch</Export-Package>
                    </instructions>
                </configuration>
```